pushapkscript/docker.d/worker.yml

```diff
@@ -127,15 +134,35 @@ products:
           skip_check_multiple_locales: true
           skip_check_same_locales: true
           skip_checks_fennec: true
-          override_channel_model: "single_google_app"
-          app:
-            package_names:
-              - "org.mozilla.fenix"
-              - "org.mozilla.fenix.beta"
-              - "org.mozilla.fenix.nightly"
-            service_account: "dummy"
-            credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FENIX_DEP_PATH" }
-            certificate_alias: "fenix"
+          apps:
+            fennec-nightly:
+              package_names: [ "org.mozilla.fennec_aurora" ]
+              certificate_alias: 'fennec-nightly'
+              google:
+                default_track: 'alpha'
+                service_account: 'dummy'
+                credentials_file: { "$eval": "GOOGLE_CREDENTIALS_FENIX_DEP_PATH" }
+            nightly:
+              package_names: [ "org.mozilla.fenix.nightly" ]
+              certificate_alias: 'fenix-nightly'
+              google:
+                default_track: 'beta'
```

Comment:
Somewhat confusing terminology for `nigtly` to be mapped to `beta` and `beta` to be mapped to` beta-closed`. But since this is staging anyway, I suppose it doesn't matter much.

Lated edit: nvm, I noticed it's the same for Fenix too. I guess that's the way we established the terminology in GP at the beginning 👍

Reply:
I agree, it's confusing 😅 

On the bright side, this migration will make things clearer eventually!